src/builder/command.rs

```diff
@@ -298,10 +298,10 @@ impl<'help> App<'help> {
     /// assert!(res.is_ok());
     /// ```
     #[must_use]
-    pub fn mut_subcommand<T, F>(mut self, subcmd_id: T, f: F) -> Self
+    pub fn mut_subcommand<'a, T, F>(mut self, subcmd_id: T, f: F) -> Self
     where
         F: FnOnce(App<'help>) -> App<'help>,
-        T: Into<&'help str>,
```

Comment:
Huh, I had assumed that the command's name was borrowed and not owned.

Reply:
yeah I found it kind of strange that `Command::new` uses `Into<String>` but `Arg::new` uses `Into<&'help str>` but 🤷